src/webcam/video.ts

```diff
@@ -0,0 +1,65 @@
+const MediaRecorder = window.MediaRecorder
+
+const handleDataAvailable = (event: BlobEvent, recordedBlobs: Array<Blob>) => {
+  if (event.data && event.data.size > 0) {
+    recordedBlobs.push(event.data)
+  }
+}
+
+const handleStop = (event: Event) => {
+  console.log('Recorder stopped: ', event)
```

Comment:
Any reason we have this? Can we wrap if so it logs for development only.

Reply:
no real reason, I copy pasted as is. Updated